layouts/shortcodes/paypal.html

```diff
@@ -1,4 +1,6 @@
 <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
+  <input type="hidden" name="country" value="{{ .Page.Language.Params.paypalCountry }}" />
+  <input type="hidden" name="lc" value="{{ .Page.Language }}" />
```

Comment:
To get to correct value (cf #804) I think you want:
```suggestion
  <input type="hidden" name="lc" value="{{ .Page.Language.Params.languageCode }}" />
```